### ISSUE TITLE
Add ding sound when changing ratio causes in proportion state

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#178]
From design meeting today, changing the number pickers (second screen) or combo box (first screen) to an inproportion ratio state should ding.

**Comment by zepumph** [comment]
Implemented above. This was done by allowing the InProportion sound to play whenever the model is in that state, no matter the cause. This also means that it will always play fully until the model is no longer in proportion.  The two main places where this can be weird are:

1. The long sound can bleed into another screen if you change screens while it plays. I tested out the "LongSoundTest" in the tambo demo and witnessed the same behavior. I also asked @jbphet about it.
2. Now "releasing" the hands while in proportion won't stop the sound, so you may get many more full, unclipped sound plays when the hands are brought to the target ratio.

@emily-phet signed off on these changes for tomorrow's interview, but I wasn't totally sure that this was a permanent sign off. Over to her for final review.

**Comment by zepumph** [comment]
@jbphet mentioned on slack that one way to solve (1) is:
> by using the `associatedViewNode` option in the `addSoundGenerator` method.

Good to keep in  mind that there is an easy solution.

**Comment by zepumph** [comment]
The sound design team discussed this at length today, in regards to https://github.com/phetsims/ratio-and-proportion/issues/174#issuecomment-696472593. We think that we should revert this commit, and tie the in-proportion sound only to user interaction of the hands. We feel that hearing the sound as part of the target ratio changing doesn't support the visual display and the description display in the same way, as sound is really supplemental to both. 

I'll revert it.

**Comment by zepumph** [comment]
Reverted above, tagging #174 as this may want review over there.
